beast/fitting/tests/test_fit_grid.py

```diff
@@ -0,0 +1,200 @@
+import os.path
+import filecmp
+import pytest
+
+import numpy as np
+
+from astropy.table import Table
+from astropy.io import fits
+from astropy.utils.data import download_file
+from astropy.tests.helper import remote_data
+from astropy import units
+
+from beast.observationmodel.observations import Observations
```

Comment:
Assuming beast is installed, correct?

Reply:
When running tests (with 'python setup.py tests' or via the travis service) the beast is first installed, then the tests are run.  So this is as it should be.